UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment/PosFulfilmentJob.cs

```diff
@@ -27,10 +27,11 @@ public async Task ProcessFulfilmentJob()
                 string result = await _fulfilmentDataService.CreatePosExchangeSets();
 
                 _logger.LogInformation(EventIds.PosFulfilmentJobCompleted.ToEventId(), "Periodic Output Service webjob completed at {DateTime} | _X-Correlation-ID : {CorrelationId}", DateTime.Now.ToUniversalTime(), CommonHelper.CorrelationID);
+                Console.ReadLine();
```

Comment:
I assume this is to keep the window open once finished for review. Does this have an impact when running in Azure?

Reply:
Removed